app/policies/aggregation_policy.rb

```diff
@@ -19,7 +19,7 @@ def resolve(action)
   end
 
   scope :index, :show, with: ReadScope
-  scope :update, :destroy, :update_links, :destroy_links, :versions, :version, with: WriteScope
+  scope :update, :destroy, :versions, :version, with: WriteScope
```

Comment:
As I understand, this is going away because it doesn't support linking in the routes. It was listed in the `can_through_parent` call before, but as I've now found out that list was meaningless and not actually referenced by anything.

Reply:
yep - those actions could have been checked against the whitelist but werent' https://github.com/zooniverse/Panoptes/blob/b4b98f9843ec759c00f5546fd4e0be3e86b633f3/lib/role_control/parental_controlled.rb#L36